config/operator.yaml

```diff
@@ -32,3 +32,10 @@ spec:
                   fieldPath: metadata.namespace
             - name: METRICS_DOMAIN
               value: knative.dev/serving-operator
+            - name: CONFIG_LOGGING_NAME
+              value: config-logging
+            - name: CONFIG_OBSERVABILITY_NAME
+              value: config-observability
```

Comment:
I do not think we need to add CONFIG_LOGGING_NAME and CONFIG_OBSERVABILITY_NAME, anymore.
By default, knative/pkg will pick config-logging and config-observability.

Reply:
It's true that knative/pkg will watch config-logging and config-observability configmaps by default if nothing is specified. Maybe it is meaningful to add here to let the users be aware of the purposes of these two configmaps.

Reply:
Though the configmap filenames are quite self-explaining, the users can not see if or where the configmaps are watched unless digging into knative/pkg

Reply:
https://github.com/knative/pkg/commit/94911510c3dc726029d2a1c0af8a5e31301eb2d7 
I just had this PR merged. If config-logging and config-observability are missing, the controller should be launched without problem. With this PR, would it be fine, if  config-logging and config-observability are missing??

Reply:
@jihuin again, you are correct here. 

See https://github.com/knative/eventing/commit/36058f4fe735902265b63ab43c3f82ff289d07ef for adding port (the `CONFIG_XYZ_NAME` were already in the yamls, eg:

https://github.com/knative/eventing/blob/master/config/channels/in-memory-channel/500-controller.yaml#L37-L40

Reply:
If the configmaps are missing and not watched, the exporter won't be set up and no metric is exported, but no information is provided in the logs. The configs here can be informative.